design.md

```diff
@@ -2,12 +2,24 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+The target was users who find it difficult to parse the original picobot language. My language is more english-adjacent such that it reads as a comprehensible sentence, almost.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
```

Comment:
I like this idea! it is very simple but at the same time tackles a challenging problem with picobot. I also felt like in cs5 i had similar issues.